__tests__/app.test.js

```diff
@@ -29,11 +29,38 @@ describe("GET/api/categories", () => {
         });
     });
   });
-
+  describe("GET//api/reviews", ()=>{
+    test('status:200, responds with an array of reviews objects has comment_count keys', () => {
+      return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((response) => {
+          const  reviews  = response.body.reviews;
+          expect(reviews ).toBeInstanceOf(Array);
+          expect(reviews ).toHaveLength(13);
+          reviews.forEach((reviews ) => {
+            expect(reviews ).toEqual(
+              expect.objectContaining({
+                review_id: expect.any(Number),
+                title: expect.any(String),
+                category: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
```

Comment:
The ticket does not ask for the review_body, you might need to be more specific with what you return with your SQL query. The body contains lots of data which isn't needed at this point. When you use a news app it doesn't display long paragraphs from the articles for you to read, it only displays the titles etc. This unnecessary data can slow things down and clog up your console.logs